.ci/esy-build-steps.yml

```diff
@@ -10,6 +10,13 @@ steps:
 
   - script: 'esy test'
     displayName: 'esy test'
+  
+  - task: PublishTestResults@2
+    displayName: 'Publish JUnit file'
+    inputs:
+      testResultsFormat: JUnit
+      testResultsFiles: './junit.xml'
```

Comment:
if you move over to using `esy dune runtest` this is a bit more involved but I have an example for how to do it.

Reply:
Thanks for bring that up, I don't think we are likely to do that, I played with it in the past and the swallowing of the incremental output was a dealbreaker for me. If you can point out the example I'd appreciate it anyway because I plan on doing some documentation about CI integration at some point.

Reply:
https://github.com/ulrikstrid/reenv/blob/master/.ci/utils/upload-test-results.yml

You basically have to save the variable first and then you use it